SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -183,8 +198,12 @@ void FSpatialGDKEditorToolbarModule::PreUnloadCallback()
 {
 	LocalReceptionistProxyServerManager->TryStopReceptionistProxyServer();
 
-	if (AutoStopLocalDeployment == EAutoStopLocalDeploymentMode::OnExitEditor)
+	if (AutoStopLocalDeployment != EAutoStopLocalDeploymentMode::Never)
 	{
+		if (InspectorProcess && InspectorProcess->Update())
```

Comment:
Explicit nullptr check for InspectorProcess

Reply:
Changed to `InspectorProcess.IsSet()` as this is a `TOptional<FMonitoredProcess>`